x-pack/plugins/lens/public/datasources/form_based/operations/definitions/formula/formula_public_api.ts

```diff
@@ -31,6 +32,7 @@ export interface FormulaPublicApi {
     column: {
       formula: string;
       label?: string;
+      filter?: Query;
```

Comment:
It is actually a good reminder to add also `timeScale` and `reducedTimeRange` here (and to not forget in the future to add them when introduced in formula op).